### DISCUSSION TITLE
can you help me? about npm run watch can't run this project

### DISCUSSION THREAD

**Discussion by cookmaster** [danwild/leaflet-velocity#81] [Q&A]
first:
npm install 
 then done:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! leaflet-velocity@1.4.4 watch: `./node_modules/gulp/bin/gulp.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the leaflet-velocity@1.4.4 watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     D:\Nodejs\node_cache\_logs\2019-08-09T07_33_17_760Z-debug.log

please call me: wellwater0327@live.cn

**Comment by danwild** [comment]
@cookmaster - what does the log say?
@86100488z - did you add a layer to the map with the leaflet layer control?

**Comment by 86100488z** [comment]
> @cookmaster - what does the log say?
> @86100488z - did you add a layer to the map with the leaflet layer control?

Problem solved,I forgot to choose layer...

**Comment by cookmaster** [comment]
Oh,i still thank you so very much.but i can't solved at now.

**Comment by cookmaster** [comment]
dear:
    hi,thank you for my problem.
   but i can't solved this question in log.




------------------ 原始邮件 ------------------
发件人: "86100488z"<notifications@github.com>;
发送时间: 2019年8月13日(星期二) 上午9:55
收件人: "danwild/leaflet-velocity"<leaflet-velocity@noreply.github.com>;
抄送: "钻石水晶"<905482164@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [danwild/leaflet-velocity] can you help me? about npm run watchcan't run this project (#38)



 
@cookmaster - what does the log say?
 @86100488z - did you add a layer to the map with the leaflet layer control?
  
Problem solved,I forgot to choose layer...

**Comment by AmirFarhan** [comment]
Have you tried running the demo?

**Comment by cookmaster** [comment]
i tried it,but can't run.



------------------ 原始邮件 ------------------
发件人: "AmirFarhan"<notifications@github.com>; 
发送时间: 2019年9月5日(星期四) 下午3:28
收件人: "danwild/leaflet-velocity"<leaflet-velocity@noreply.github.com>; 
抄送: "钻石水晶"<905482164@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [danwild/leaflet-velocity] can you help me? about npm run watchcan't run this project (#38)




Have you tried running the demo?

**Comment by cookmaster** [comment]
------------------ 原始邮件 ------------------
发件人: "AmirFarhan"<notifications@github.com>; 
发送时间: 2019年9月5日(星期四) 下午3:28
收件人: "danwild/leaflet-velocity"<leaflet-velocity@noreply.github.com>; 
抄送: "钻石水晶"<905482164@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [danwild/leaflet-velocity] can you help me? about npm run watchcan't run this project (#38)




Have you tried running the demo?

**Comment by AmirFarhan** [comment]
[#39](https://github.com/danwild/leaflet-velocity/issues/39) try referring to this and do tell if you still have any problems. It will be helpful if you can share your package.json too.

**Comment by cookmaster** [comment]
{
  "name": "leaflet-velocity",
  "version": "1.4.4",
  "description": "A custom layer for leaflet to visualise arbitrary velocities",
  "main": "dist/leaflet-velocity.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "watch": "./node_modules/gulp/bin/gulp.js",
    "gulp": "gulp"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/danwild/leaflet-velocity.git"
  },
  "keywords": [
    "leaflet",
    "velocity",
    "ocean",
    "current",
    "wind",
    "canvas",
    "noaa",
    "visualisation",
    "overlay"
  ],
  "author": "danwild@y7mail.com",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/danwild/leaflet-velocity/issues"
  },
  "homepage": "https://github.com/danwild/leaflet-velocity#readme",
  "dependencies": {},
  "devDependencies": {
    "@babel/core": "^7.5.4",
    "@babel/preset-env": "^7.5.4",
    "babel-preset-es2015": "^6.9.0",
    "cssnano": "^4.1.10",
    "gulp": "^2.2.0",
    "gulp-babel": "^8.0.0",
    "gulp-concat": "^2.6.1",
    "gulp-concat-css": "^3.1.0",
    "gulp-postcss": "^8.0.0",
    "gulp-rename": "^1.4.0",
    "gulp-uglify": "^3.0.2"
  }
}

**Comment by cookmaster** [comment]
it's no change

**Comment by cookmaster** [comment]
at least ,thank you very much.

**Comment by AmirFarhan** [comment]
It seems like you did not load jquery and leaflet as your dependencies. Do the following in your repo:
npm install jquery
npm install leaflet

**Comment by cookmaster** [comment]
wow,i try it,thank you

**Comment by cookmaster** [comment]
can't  run,the same problem in the first

**Comment by brentfraser** [comment]
If you are running node js  on Windows, edit the package.json file and change line 8 from:

`    "watch": "./node_modules/gulp/bin/gulp.js"``
`
to

`    "watch": "node ./node_modules/gulp/bin/gulp.js"`
